docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -1,10 +1,10 @@
 ---
 productId: x-date-pickers
-title: Date and Time Pickers - Custom subcomponents
+title: Date and Time Pickers - Custom slots and subcomponents
 components: DateTimePickerTabs
```

Comment:
WDYT about updating the list of relevant components here to sync with the actual demos that we currently have? 🤔 
```suggestion
components: DateTimePickerTabs, PickersActionBar, DatePickerToolbar, TimePickerToolbar, DateTimePickerToolbar, PickersCalendarHeader, PickersShortcuts
```